DIGIT210/classwork/akam-python-test.py

```diff
@@ -46,16 +48,16 @@ def entitycollector(tokens):
 
 config = {"spans_key": None, "annotate_ents": True, "overwrite": True, "validate": True}
 ruler = nlp.add_pipe("span_ruler", before="ner", config=config)
-def readTextFiles(filepath):
-    # with open(filepath, 'r', encoding='utf8') as f:
+def readTextFiles(souls):
```

Comment:
@tylerakam This is the `readTextFiles()` function that is finally working now in your file!